.github/workflows/pages.yml

```diff
@@ -30,7 +30,8 @@ jobs:
           distribution: 'liberica'
       - name: Generate website (XSLT)
         id: generate
-        run: mvn -B package -Pquick-start-guide-website
+        # Override Maven properties for the Petal button url
```

Comment:
@plutonik-a Can you remove the trailing `/` 's please? They should be computed in the XSLT.

Reply:
Fixed with 5517d0c